### PR TITLE
added Java Toolkits

### PR DESCRIPTION
**Pull request by Ablesius**
Apache OpenNLP
Stanford CoreNLP

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,10 +51,10 @@ If you want to contribute to this list (please do), send a pull request or conta
 <a name="Text Mining" />
 ## Text Mining
 
-<a name="Python" />
-#### Python
-* [NLTK](http://www.nltk.org/) - Natural language toolkit.
-* [Orange](http://orange.biolab.si/) - Open source data visualization and data analysis for novice and expert. Interactive workflows with a large toolbox.
+<a name="Java" />
```
Comment (ajdapretnar) [inline README.md]: Could you please not delete lines? These are proper information that will stay in the list. Please fix this and I'll merge.

Comment (Ablesius) [inline README.md]: Oops, that was a mistake actually! My bad, sorry. Will be fixed in a minute!

**Comment by Ablesius** [comment]
fixed. Sorry again.

**Comment by ajdapretnar** [comment]
No worries. I'm merging. Thanks for the PR!

**Pull request merged**